test/kubernetes/e2e/k8sgateway/k8s_gw_test.go

```diff
@@ -57,4 +58,8 @@ func TestK8sGateway(t *testing.T) {
 	t.Run("RouteOptions", func(t *testing.T) {
 		suite.Run(t, route_options.NewTestingSuite(ctx, testInstallation))
 	})
+
+	t.Run("Port Routing", func(t *testing.T) {
```

Comment:
https://github.com/solo-io/gloo/blob/main/test/kubernetes/e2e/example/error_logging_test.go#L51
```suggestion
	t.Run("PortRouting", func(t *testing.T) {
```

Reply:
Related to this- do you have any suggestions for a better name for the "feature" that "PortRouting"?